### DISCUSSION TITLE
MindTheGapps not installing correctly

### DISCUSSION THREAD

**Discussion by eadmaster** [openandroidinstaller-dev/openandroidinstaller#527] [Q&A] [answered]
i've managed to install `lineage-17.1-20220915-UNOFFICIAL-dreamlte` + `MindTheGapps-10.0.0-arm64-20230922_081111` on an old Samsung S8 after a few tries.
The installation went fine without errors but i don't see the Google Apps installed.

[openandroidinstaller.log](https://github.com/openandroidinstaller-dev/openandroidinstaller/files/15284140/openandroidinstaller.log)

**Comment by eadmaster** [comment, marked as answer]
update: fixed by installing an old version of `open_gapps-arm64-10.0-stock-20220215` from https://opengapps.org/

**Reply by tsterbak** [reply]
Nice that it worked!
